src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -385,7 +452,7 @@ public void uploadBdioEntries(BlackDuckRunData blackDuckRunData, UUID bdScanId)
         BlackDuckServicesFactory blackDuckServicesFactory = blackDuckRunData.getBlackDuckServicesFactory();
         BlackDuckApiClient blackDuckApiClient = blackDuckServicesFactory.getBlackDuckApiClient();
 
-        String contentType = "application/vnd.blackducksoftware.scan-evidence-1+protobuf";
+        String contentType = RAPID_SCAN_CONTENT_TYPE;
         HttpUrl putUrl = new HttpUrl(blackDuckRunData.getBlackDuckServerConfig().getBlackDuckUrl().toString()
```

Comment:
Use a final field for the literal. Use .concat to the join the strings.

Reply:
Removed that variable as we can use RAPID_SCAN_CONTENT_TYPE directly. 
Using .concat is good. I just found out that the HttpUrl class from integration-rest provides an `appendRelativeUrl` method to join 2 paths. This is better than concat as it takes care of trailing/leading slashes while joining 2 paths.